### DISCUSSION TITLE
whats a good minimal and very simple blockly html template i can easily edit?

### DISCUSSION THREAD

**Discussion by rubiidev18alt** [cytrinc/bark-coding#162] [Q&A]
I wanna make my own blockly mod inspired by bark, it might just be a blockly editor page with just the blockly thing and a homepage because I might be very lazy on it.

you can help too! Just like in bark.

(btw the projects name I gonna be "hoot" and the mascot is an owl)

(the orginization: https://github.com/Hoot-Coding)

(also I'm testing out teams in this so join what team you think is best for you.)

**Comment by nmsderp** [comment]
blockly mods don't exist, blockly is library

**Reply by raynepaws** [reply]
i think they mean something like bark is doing

**Reply by mariocraft987** [reply]
yes

**Comment by mariocraft987** [comment]
Seems like a spinoff of my failed [Owl Blockly](https://github.com/mariocraft987/owl-blockly)
